miden-tx/src/tests/kernel_tests/test_epilogue.rs

```diff
@@ -1,67 +1,71 @@
-use alloc::{string::String, vec::Vec};
+use alloc::vec::Vec;
 
-use miden_lib::transaction::{
-    memory::{CREATED_NOTE_ASSET_HASH_OFFSET, CREATED_NOTE_SECTION_OFFSET, NOTE_MEM_SIZE},
-    ToTransactionKernelInputs,
+use miden_lib::transaction::memory::{
+    CREATED_NOTE_ASSET_HASH_OFFSET, CREATED_NOTE_SECTION_OFFSET, NOTE_MEM_SIZE,
 };
-use miden_objects::testing::notes::AssetPreservationStatus;
-use vm_processor::ProcessState;
-
-use super::{
-    build_module_path, output_notes_data_procedure, ContextId, MemAdviceProvider, TX_KERNEL_DIR,
-    ZERO,
+use miden_objects::{
+    accounts::{account_id::testing::ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_OFF_CHAIN, Account},
+    transaction::{OutputNote, OutputNotes},
 };
-use crate::testing::{executor::CodeExecutor, utils::mock_executed_tx};
-
-const EPILOGUE_FILE: &str = "epilogue.masm";
-
-/// Loads epilogue file and returns the complete code formatted as
-/// "{imports}{epilogue_code}{code}"`
-#[cfg(feature = "std")]
-fn insert_epilogue(imports: &str, code: &str) -> String {
```

Comment:
I'm not sure what this was trying to accomplish. Replaced it with a simpler `use.miden...` instead.